evm_responses.yaml

```diff
@@ -828,7 +827,7 @@ transfer:
     erc721TokenId:
       type: string
       nullable: true
-      description: 'Raw ERC721 token id (hex string). null if not an ERC721 token transfer.'
+      description: 'Legacy token ID field for ERC721 tokens (hex string). The `tokenId` field should be used instead.'
```

Comment:
Perhaps you can add `(DEPRECATED)` to the beginning of the description to be extra clear

Reply:
Done!